specialist-publisher/config/deploy.rb

```diff
@@ -7,5 +7,13 @@
 load 'deploy/assets'
 load 'govuk_admin_template'
 
+namespace :deploy do
+  desc "Publish all Finders to the Publishing API"
+  task :publish_finders do
+    run "bundle exec rake publishing_api:publish_finders"
```

Comment:
This won't work, you need `govuk_setenv` and more. Maybe check how other things in this repo run rake tasks.